include/ros_sec_test/attacks/periodic_attack_component.hpp

```diff
@@ -36,23 +36,25 @@ namespace attacks
 class PeriodicAttackComponent : public rclcpp_lifecycle::LifecycleNode
 {
 public:
+  PeriodicAttackComponent();
```

Comment:
We should try to avoid changing the class interface to enable testing. Could we remove this?